localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -84,24 +84,23 @@ NDTScanMatcher::NDTScanMatcher()
   tf2_broadcaster_(*this),
   ndt_ptr_(new NormalDistributionsTransform),
   state_ptr_(new std::map<std::string, std::string>),
-  base_frame_("base_link"),
-  ndt_base_frame_("ndt_base_link"),
-  map_frame_("map"),
-  converged_param_type_(ConvergedParamType::TRANSFORM_PROBABILITY),
-  converged_param_transform_probability_(4.5),
-  converged_param_nearest_voxel_transformation_likelihood_(2.3),
-  initial_estimate_particles_num_(100),
-  lidar_topic_timeout_sec_(1.0),
-  initial_pose_timeout_sec_(1.0),
-  initial_pose_distance_tolerance_m_(10.0),
-  inversion_vector_threshold_(-0.9),
-  oscillation_threshold_(10),
+  base_frame_(),
+  ndt_base_frame_(),
+  map_frame_(),
+  converged_param_type_(),
+  converged_param_transform_probability_(),
+  converged_param_nearest_voxel_transformation_likelihood_(),
+  initial_estimate_particles_num_(),
+  lidar_topic_timeout_sec_(),
+  initial_pose_timeout_sec_(),
+  initial_pose_distance_tolerance_m_(),
```

Comment:
Are these lines necessary? I m not sure how they will work.

Reply:
Actually, you're right!
Since they are defined in the hpp file, and they soon initialize their value from `ndt_scan_matcher.param.yaml`, I removed the parameters that have empty brackets.

I checked that I can colcon build ndt_scan_matcher, and logging simulator tutorial works fine.

Reply:
Thanks.  :+1: